### PR TITLE
Remember the full user data on login

### PR DESCRIPTION
**Pull request by ochriste**
for example to allow storing json comming from a LDAP authentication.

### CODE REVIEW

File: doc/integrator/authentication.rst [hunk 1]
```diff
@@ -210,3 +213,108 @@ LDAP would look like this::
...
+            authentication_policy=create_authentication(settings)
+        )
+
+        config.include("c2cgeoportal")
+        config.include('pyramid_ldap')
+
+        config.ldap_setup(
+            'ldap://ldap.server.host',
+            bind='CN=ldap user,CN=Users,DC=example,DC=com',
+            passwd='ld@pu5er'
```
Comment (sbrunner) [inline doc/integrator/authentication.rst]: Put a fake password ...

Comment (ochriste) [inline doc/integrator/authentication.rst]: thats the fake password from pyramid_ldap http://docs.pylonsproject.org/projects/pyramid-ldap/en/latest/

File: doc/integrator/authentication.rst [hunk 2]
```diff
@@ -210,3 +213,108 @@ LDAP would look like this::
...
+        # add the interfaces
+        add_interface(config)
+        add_interface(config, "edit")
+        add_interface(config, "routing")
+        add_interface(config, "mobile", INTERFACE_TYPE_SENCHA_TOUCH)
+
+        config.set_user_validator(custom_user_validator)
+        config.add_request_method(custom_get_user_from_request, 'user', reify=True)
+
+        return config.make_wsgi_app()
```
Comment (sbrunner) [inline doc/integrator/authentication.rst]: I think that better is you put only the specific code ...

File: doc/integrator/authentication.rst (hunk 1 above)
Comment (sbrunner) [inline doc/integrator/authentication.rst]: ok :-/

File: doc/integrator/authentication.rst [hunk 3]
```diff
@@ -180,19 +180,22 @@ For logging in ``c2cgeoportal`` validates the user credentials
 table. If a c2cgeoportal application should work with another user information
 source, like LDAP, another *client validation* mechanism should be set up.
 ``c2cgeoportal`` provides a specific ``Configurator`` function for that, namely
-``set_user_validator``. Here's an example::
+``set_user_validator`` which allow to register a custom validator.
+Here's an example::
 
-    def user_validator(request, username, password):
```
Comment (sbrunner) [inline doc/integrator/authentication.rst]: use:
`.. code:: python`

File: doc/integrator/authentication.rst [hunk 4]
```diff
@@ -202,7 +205,7 @@ User validators can obviously be chained. For example, a user validator
 function that queries the ``user`` database table if the user does not exist in
 LDAP would look like this::
 
-    def user_validator(request, username, password):
```
Comment (sbrunner) [inline doc/integrator/authentication.rst]: same

**Comment by sbrunner** [comment]
Just some comment else that's looks good :-)

**Pull request merged**